### ISSUE TITLE
ExportVMHostConfiguration - DependsOn quoting error

### ISSUE THREAD

**Issue by lucdekens** [vmware/dscr-for-vmware#257]
When a VSS depends on a number of pNIC, the generated code looks like
```powershell
DependsOn = '[VMHostPhysicalNic]VMHostPhysicalNic_vmnic1, [VMHostPhysicalNic]VMHostPhysicalNic_vmnic0'
```

while that should be

```powershell
DependsOn = '[VMHostPhysicalNic]VMHostPhysicalNic_vmnic1', '[VMHostPhysicalNic]VMHostPhysicalNic_vmnic0'
```

This causes an error when running the config.

![image](https://user-images.githubusercontent.com/7760649/76261675-cfbbee00-625a-11ea-93a6-eab7518d2896.png)

**Comment by SimeonGerginov** [comment]
Good catch! I will start working on fixing it.

**Comment by SimeonGerginov** [comment]
The bug was fixed in PR #260.
